src/Illuminate/Validation/Validator.php

```diff
@@ -216,7 +216,7 @@ protected function explodeRules($rules)
     {
         foreach ($rules as $key => $rule) {
             if (Str::contains($key, '*')) {
-                $this->each($key, $rule);
+                $this->each($key, [$rule]);
```

Comment:
First I tried to change the `for` statement in the `each()` method, but as it is a public method I was afraid of breaking something somewhere else.

Just to be clear regarding your second comment, if `$rule` was already an array that was the case that was failing.